### PR TITLE
WCM learning path edits

### PR DESCRIPTION
**Pull request by ngaskill**
This includes my latest edits, through 03-creating-and-styling-content/04-publishing-content.markdown (which also includes the sub-section on structures and templates). Also includes Rich's edits that he sent me in https://github.com/ngaskill/liferay-docs/pull/142

@stephenkostas

### CODE REVIEW

**Comment by mwilliams2014** [comment]
@sez11a I'll wait for you to complete your review before I add the UI to this section, just in case anything changes. Just ping me when it's ready to go. Thanks!

**Comment by sez11a** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: discover/portal/articles/001-content-sites/03-creating-and-styling-content/00-creating-content-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+So far you've created the Lunar Resort site's outline. By creating pages and
+using templates, the site now has a structure and the pages have layouts. Now
+it's time to color everything in with content.
+
+In this section you'll create content, some with more complex web content
+structures and templates, and use Liferay's Web Experience Management to publish
+your new content on your site.
+
+![Figure x: IMAGE.](../../images/.png)
```
Comment (sez11a) [inline discover/portal/articles/001-content-sites/03-creating-and-styling-content/00-creating-content-intro.markdown]: This is still missing the image. Anybody got an image they can commit? @stephenkostas @ngaskill

File: discover/portal/articles/001-content-sites/03-creating-and-styling-content/03-structures-and-templates/00-structures-and-templates-intro.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The template can contain CSS, HTML, Javascript, and elements of the templating 
+language it uses. 
+
+Before getting started with structures and templates, you should know something 
+important--you might want to sit down for this--you've actually already used 
+them! Basic Web Content is a type of web content defined by a structure and a 
+template. Now you'll use structures and templates to create something a little 
+more intricate. 
+
+![Figure x: IMAGE](../../../../images/.png)
```
Comment (sez11a) [inline discover/portal/articles/001-content-sites/03-creating-and-styling-content/03-structures-and-templates/00-structures-and-templates-intro.markdown]: @stephenkostas @ngaskill Is an image forthcoming?

**Comment by sez11a** [comment]
Superseded by https://github.com/stephenkostas/liferay-docs/pull/37

**Pull request closed without merging**